packages/neuron-ui/src/widgets/Calendar/index.tsx

```diff
@@ -67,8 +67,8 @@ const Calendar: React.FC<CalendarProps> = ({
   firstDayOfWeek = 0,
   className = '',
 }) => {
-  const [year, setYear] = useState(new Date().getFullYear())
-  const [month, setMonth] = useState(new Date().getMonth() + 1)
+  const [year, setYear] = useState(value?.getFullYear() ?? new Date().getFullYear())
+  const [month, setMonth] = useState((value?.getMonth() ?? new Date().getMonth()) + 1)
```

Comment:
Should be `value?.getFullYear` and `value?.getMonth`?

Reply:
This is to address a previously existing bug where the panel did not automatically navigate to the correct position when a date attribute was passed.
After giving it some thought, I have come up with a more appropriate fix.
PR updated.